src/fix_calibration.R

```diff
@@ -0,0 +1,389 @@
+#' @title Correct data with underlying bad calibrations using In-Situ calibration reports
+#'
+#' @description This function uses calibration reports collected by field crews to "correct"
+#' instances of poor calibration. These are infrequent, and this function is not yet complete.
```

Comment:
Question: This says the function is not yet complete, is it incorporated into the pipeline? I think that a portion of the `add_malfunction_flag.R` references this function and says that its made redundant by this function.